File: src/main/java/upbrella/be/rent/repository/LockerRepository.java
package upbrella.be.rent.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import upbrella.be.rent.entity.Locker;

import java.util.Optional;

public interface LockerRepository extends JpaRepository<Locker, Long> {

    Optional<Locker> findByStoreMetaId(Long storeMetaId);
}

File: src/main/kotlin/upbrella/be/rent/repository/LockerRepository.kt
package upbrella.be.rent.repository

import org.springframework.data.jpa.repository.JpaRepository
import upbrella.be.rent.entity.Locker
import java.util.*

interface LockerRepository : JpaRepository<Locker, Long> {

    fun findByStoreMetaId(storeMetaId: Long): Optional<Locker>
}